packages/benchmark-wasm/benchmarks/folding_ranges.bench.ts

```diff
@@ -40,15 +40,19 @@ h4 {
 `;
 
 const textDocument = TextDocument.create("file:///test.css", "css", 0, content);
+const ls = new LanguageService({
+	include_base_css_custom_data: true,
+});
+
+await ls.upsert_document(textDocument);
 
 describe("Folding Ranges", async () => {
 	bench("CSSLSRS(WASM) - Folding Ranges", async () => {
-		await get_folding_ranges(textDocument);
+		await ls.get_folding_ranges(textDocument);
 	});
 	if (!process.env.CODSPEED) {
 		bench("vscode-css-languageservice - Folding Ranges", () => {
-			const stylesheet = vscodeLanguageService.parseStylesheet(textDocument);
-			vscodeLanguageService.getFoldingRanges(textDocument, stylesheet);
+			vscodeLanguageService.getFoldingRanges(textDocument);
```

Comment:
The benchmark here was actually inaccurate, VSC's implementation does not use a stylesheet here. The difference between the two is still drastic, but not as much.